cdap-docs/developers-manual/source/building-blocks/namespaces.rst

```diff
@@ -28,8 +28,9 @@ namespaces include:
 The first version of namespaces was introduced in :ref:`CDAP v2.8.0 <release-notes>`, and
```

Comment:
Not part of this PR, but in lines 24-26 above, should we call it 'Partitioning a single CDAP cluster'